webapp-django/crashstats/crashstats/templates/crashstats/crashes_per_user.html

```diff
@@ -0,0 +1,275 @@
+                <tr>
+                    <th class="date" rowspan="2">Date</th>
+                  {% for version in versions %}
+                    <th class="version" colspan="4">{{ version }}</th>
+                  {% endfor %}
+                </tr>
+                <tr>
+                  {% for version in versions %}
+                    <th class="stat">Crashes</th>
+                    <th class="stat">ADI</th>
+                    <th class="stat throttle">Throttle</th>
+                    <th class="stat">Ratio</th>
+                  {% endfor %}
+                </tr>
+                {% for crash_date in dates %}
+                <tr>
+                  <td>{{ crash_date }}</td>
+                    {% if crash_date in data_table.dates %}
+                      {% for version in versions %}
+                        {% set foundversion = [] %}
+                        {% for crash_info in data_table.dates[crash_date] %}
+                          {% if crash_info.version == version %}
+                      <td>{{ crash_info.report_count | digitgroupseparator }}</td>
+                      <td>{{ crash_info.adi | digitgroupseparator }}</td>
+                      <td class="throttle"
+                      >{% if crash_info.throttle %}{{ crash_info.throttle * 100 }}%{% else %}-{% endif %}</td>
+                      <td>{{ crash_info.ratio }}</td>
+                            {% do foundversion.append(1) %}
+                          {% endif %}
+                        {% endfor %}
+                        {% if not foundversion %}
+                      <td>-</td>
+                      <td>-</td>
+                      <td class="throttle">-</td>
+                      <td>-</td>
+                        {% endif %}
+                      {% endfor %}
+                    {% else %}
+                      {% for version in versions %}
+                      <td>-</td>
+                      <td>-</td>
+                      <td class="throttle">-</td>
+                      <td>-</td>
+                      {% endfor %}
+                    {% endif %}
+                </tr>
+                {% endfor %}
+            </table>
+          {% else %}
+          <p>
+            No data is available for this query.
+          </p>
+          {% endif %}
+        </div>
+    </div>
+</div>
+{% endblock %}
+{% block site_js %}
+  {{ super() }}
+
+  {% compress js %}
+  <script src="{{ static('crashstats/js/jquery/plugins/jquery-ui.js') }}"></script>
+  {% endcompress %}
+
+  {% compress js %}
+  <script src="{{ static('crashstats/js/lib/d3.min.js') }}"></script>
+  <script src="{{ static('crashstats/js/lib/metricsgraphics.min.js') }}"></script>
+  <script src="{{ static('crashstats/js/socorro/crashes_per_user.js') }}"></script>
+  {% endcompress %}
+  <script>
+    var DATA = {{ graph_data | json_dumps }};
+    window.socGraphByReportType = false;
```

Comment:
If this is always false, I'm quite sure it can be removed and the JS file can be simplified.